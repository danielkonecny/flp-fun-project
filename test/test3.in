A,B
a,b
A
A->BaBaBa
A->BABABA
B->b
