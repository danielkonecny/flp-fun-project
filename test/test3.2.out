A,B,<aBaBa>,<BaBa>,<aBa>,<ABABA>,<BABA>,<ABA>,<BA>,a',<Ba>
a,b
A
A->B<aBaBa>
<aBaBa>->a'<BaBa>
<BaBa>->B<aBa>
<aBa>->a'<Ba>
<Ba>->Ba'
A->B<ABABA>
<ABABA>->A<BABA>
<BABA>->B<ABA>
<ABA>->A<BA>
<BA>->BA
B->b
a'->a
